commit f1201b890e510d0f071177fb7b39d0d54c0eb372 (refs/changes/31/4631/1)
Author: Angela Fong <angela.fong@joyent.com>
Date:   2018-08-03T11:42:17-07:00 (1 year, 2 months ago)
    
    TRITON-664: Firewall rules: update cloudapi and fwapi docs to include more examples and mention about fwrule_version

diff --git a/docs/index.md b/docs/index.md
index 33acc22..a69cfcd 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -10,7 +10,7 @@ apisections: Rules, Firewalls
 -->
 
 <!--
-    Copyright (c) 2015, Joyent, Inc.
+    Copyright (c) 2018, Joyent, Inc.
 -->
 
 # FWAPI (Firewall API)
@@ -42,7 +42,6 @@ For more information about the implementation and various pieces of the
 firewalling system, consult the
 [architecture and troubleshooting guide](architecture.html).
 
-See also the [examples](examples.html) for some common use-cases.
 
 ## Default Policy
 
@@ -124,18 +123,16 @@ There are two more steps necessary:
 * Updating all webserver VMs with tag **www**
 * Updating all database VMs with tag **db**
 
-Please see the [examples](examples.html) page for more examples.
+Please see the [CloudAPI Guide](https://github.com/joyent/sdc-cloudapi/blob/master/docs/index.md#firewallrules)
+for more examples.
 
 # Rule Syntax
 
-Rules are written in a Doman Specific Language (DSL). References for it are:
-
-* The [rule syntax guide](rules.html) has the complete syntax.
-* The [railroad diagram](railroad-diagram.xhtml) has an easier to read
-  format.
-* The [cloudapi firewall rules documentation](http://wiki.joyent.com/wiki/display/jpc2/Firewall+Rules)
-  is probably much clearer.
+Rules are written in a Doman Specific Language (DSL). References for the rule syntax
+can be found in:
 
+* [Cloud Firewall Rules Reference](https://docs.joyent.com/public-cloud/network/firewall/cloud-firewall-rules-reference)
+* [CloudAPI Guide - Firewall Rules](https://github.com/joyent/sdc-cloudapi/blob/master/docs/index.md#firewallrules)
 
 # Rules
 
@@ -473,3 +470,16 @@ something).
 ## 2015-03-23
 
 - Added `fields` option to the [ListRules](#ListRules) endpoint.
+
+## 2017-01-13
+
+- Migrated firewall rules from ufds to moray bucket to support the use
+  of certain special characters in tag names. (requires SAPI metadata
+  `FWRULE_VERSION` 3 or above)
+
+## 2018-03-16
+
+- Added `priority` support for firewall rules. (requires SAPI metadata
+  `FWRULE_VERSION` 4 or above, as well as firewaller and platform
+  image updates to include the changes in [TRITON-75](https://smartos.org/bugview/TRITON-75)
+  and [TRITON-637](https://smartos.org/bugview/TRITON-637))
